### DISCUSSION TITLE
text box dont't work well

### DISCUSSION THREAD

**Discussion by EM1994** [grafana/grafana#52383] [General]
I want to implement a complex lucene query through text box，like  : DestCityName:Torino AND Carrier:JetBeats
![image](https://user-images.githubusercontent.com/32710096/179464023-eae4e5e1-c804-4316-ac38-303be6262de2.png)
![image](https://user-images.githubusercontent.com/32710096/179464139-971e77e6-f357-410e-8672-e0513f2c0f80.png)
but it seem don't work 
1) input DestCityName:Torino AND Carrier:JetBeats in text box named test
![image](https://user-images.githubusercontent.com/32710096/179464457-01c3d7e5-afff-4d85-aa77-0873211446f1.png)
2)  Query inspector show like flow, why sapce and :  were be escaped ? I dont't understand  :)  
![image](https://user-images.githubusercontent.com/32710096/179464602-bf7f93a6-a0e5-4226-8cbd-a47ef5493748.png)

Because of these escapes, es doesn't recognize this query
"{\"search_type\":\"query_then_fetch\",\"ignore_unavailable\":true,\"index\":\"kibana_sample_data_flights\"}
{\"size\":500,\"query\":{\"bool\":{\"filter\":[{\"range\":{\"timestamp\":{\"gte\":1658112469152,\"lte\":1658134069152,\"format\":\"epoch_millis\"}}},{\"query_string\":{\"analyze_wildcard\":true,\"query\":\"DestCityName\\\\:Torino\\\\ AND\\\\ Carrier\\\\:JetBeats\"}}]}},\"sort\":[{\"timestamp\":{\"order\":\"desc\",\"unmapped_type\":\"boolean\"}},{\"_doc\":{\"order\":\"desc\"}}],\"script_fields\":{}}

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
